ui/helpers/url.js

```diff
@@ -77,24 +77,42 @@ export const getProjectJobUrl = function getProjectJobUrl(url, jobId) {
 export const linkifyURLs = function linkifyURLs(input) {
   const urlpattern = /(\b(https?):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gim;
 
-  return input.replace(urlpattern, '<a href="$1" target="_blank" rel="noopener">$1</a>');
```

Comment:
I can't find a way to do ``target="_blank"` with Markdown, but I don't think it's terribly important.  I guess we'll see if people complain.  :)